### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by 17thSep**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by 17thSep** [comment]
Hi, Please add these boilerplate projects for WDIO 7

**Comment by 17thSep** [comment]
@christian-bromann what do you think?

**Comment by 17thSep** [comment]
Thanks for ignoring.

**Comment by christian-bromann** [comment]
@17thSep this PR is pending for 4 days only. We haven't ignored it we just haven't had time to review it. There were other things that had higher priorities.

**Comment by 17thSep** [comment]
> is PR is pending for 4 days only. We haven't ignored it we just haven't had time to review it. There were other things that had higher priorities.

Apologies! Maybe I got worked up for no reason. I was just thinking from my point of view didn't think about your side.
I will reopen it again. Let me know if its ok to be merged.

File: website/docs/BoilerplateProjects.md [hunk 1]
```diff
@@ -52,6 +52,20 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
...
   - Sauce Labs integration
 
+## [17thSep/WebdriverIO_Master](https://github.com/17thSep/WebdriverIO_Master)
+
+   Boilerplate project to run **WebdriverIO v7** tests for the following <br>
+   
+   [1. Cucumber](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Cucumber)               - WDIO 7 scripts with TypeScript in Cucumber Framework<br>
+   [2. Mocha](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Mocha)           - WDIO 7 scripts with TypeScript in Mocha Framework<br>
+   [3. Docker](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Docker)         - Run WDIO 7 script in Docker<br>
+   [4. Networklogs](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/NetworkLogs)     - Capture Network logs<br>
```
Comment (BorisOsipov) [inline website/docs/BoilerplateProjects.md:62]: lead to 404

File: website/docs/BoilerplateProjects.md [hunk 2]
```diff
@@ -52,6 +52,20 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
...
+   [5. BasicTypeScript](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/BasicTypeScript) - Only TypeScript<br>
+   
+   More to follow...
+
+   ### You can also [watch them created here](https://www.youtube.com/channel/UCqaDA1xslraCbam2CxuKhUw)<br>
```
Comment (BorisOsipov) [inline website/docs/BoilerplateProjects.md:67]: could you remove this? it is not related to boilerplate projects

File: website/docs/BoilerplateProjects.md [hunk 3]
```diff
@@ -52,6 +52,20 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
   - Page Object Pattern
   - Sauce Labs integration
 
+## [17thSep/WebdriverIO_Master](https://github.com/17thSep/WebdriverIO_Master)
```
Comment (BorisOsipov) [inline website/docs/BoilerplateProjects.md:55]: could we move and join it to https://github.com/webdriverio/webdriverio/pull/6425/files#diff-5047c2de25222857de23ee59f93e410374eb238d9439bd477b0b2e6c34d4ef89R270 ?

**Review by BorisOsipov — CHANGES_REQUESTED** [review]

File: website/docs/BoilerplateProjects.md (hunk 3 above)
Comment (17thSep) [reply website/docs/BoilerplateProjects.md:55]: I am not quite following, can you please explain in detail

File: website/docs/BoilerplateProjects.md (hunk 1 above)
Comment (17thSep) [reply website/docs/BoilerplateProjects.md:62]: Networklogs alone right?

File: website/docs/BoilerplateProjects.md (hunk 2 above)
Comment (17thSep) [reply website/docs/BoilerplateProjects.md:67]: ok

File: website/docs/BoilerplateProjects.md (hunk 3 above)
Comment (BorisOsipov) [reply website/docs/BoilerplateProjects.md:55]: Move this to https://github.com/17thSep/webdriverio/blob/patch-2/website/docs/BoilerplateProjects.md#17thsepmock-and-spies

Comment (17thSep) [reply website/docs/BoilerplateProjects.md:55]: Done

Comment (17thSep) [reply website/docs/BoilerplateProjects.md:55]: looks good?

**Review by BorisOsipov — APPROVED** [review]

File: website/docs/BoilerplateProjects.md [hunk 4]
```diff
@@ -253,14 +253,17 @@ Boilerplate project to run appium tests for native and mobile browser using moch
...
+[WDIO 7 scripts with TypeScript in Cucumber Framework](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Cucumber)
+[WDIO 7 scripts with TypeScript in Mocha Framework](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Mocha)
+[Run WDIO 7 script in Docker](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Docker)
+[Network logs](https://github.com/17thSep/MonitorNetworkLogs/)
+Boiler plate project for:
+     - Capture Network Logs
+     - Capture all GET/POST calls or a specific REST API
+     - Assert Request parameters
+     - Assert Response parameters
+     - Store all the response in a separate file
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:269]:
```suggestion

Boiler plate project for:

- Capture Network Logs
- Capture all GET/POST calls or a specific REST API
- Assert Request parameters
- Assert Response parameters
- Store all the response in a separate file
```

File: website/docs/BoilerplateProjects.md [hunk 5]
```diff
@@ -253,14 +253,17 @@ Boilerplate project to run appium tests for native and mobile browser using moch
...
-- Assert Request parameters
-- Assert Response parameters
-- Store all the response in a separate file
+Boilerplate project to run **WebdriverIO v7** tests for the following:
 
-You can watch it created [on this video](https://www.youtube.com/watch?v=Be9IPyxHmLs).
+[WDIO 7 scripts with TypeScript in Cucumber Framework](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Cucumber)
+[WDIO 7 scripts with TypeScript in Mocha Framework](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Mocha)
+[Run WDIO 7 script in Docker](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Docker)
+[Network logs](https://github.com/17thSep/MonitorNetworkLogs/)
```
Comment (christian-bromann) [inline website/docs/BoilerplateProjects.md:263]: This doesn't seem to be within the other repositories, maybe split it?

Comment (BorisOsipov) [reply website/docs/BoilerplateProjects.md:263]: I think it doesn't matter.

Comment (christian-bromann) [reply website/docs/BoilerplateProjects.md:263]: fine

**Review by christian-bromann — APPROVED** [review]
👍

File: website/docs/BoilerplateProjects.md (hunk 5 above)
Comment (17thSep) [reply website/docs/BoilerplateProjects.md:263]: I am not quite following Christian, what I was thinking is to have a single repository for all that I am planning to work on and create separate folders in it to segregate. That way I don't have to update multiple repositories separately making this page so long to scroll. 
Are you asking me to create different sections for each of the folders here?

**Comment by 17thSep** [comment]
Thanks, Both. somehow I feel like a 5 yr old boy ;) Nevertheless, it's an experience.

**Pull request merged**